client/js/app/components/explorer/query_builder/index.js

```diff
@@ -199,7 +200,7 @@ var QueryBuilder = React.createClass({
   render: function() {
     var apiQueryUrl;
     if (this.props.model.isValid) {
-      apiQueryUrl = ExplorerUtils.getApiQueryUrl(this.props.client, this.props.model);
+      apiQueryUrl = new ApiQueryUrl(this.props.client, this.props.model).getUrl();
```

Comment:
Regarding my comment below about the ApiQueryUrl prototype: If you exported a single function this would become something along the lines of: `apiQueryUrl = GetApiQueryUrl(this.props.client, this.props.model);`